src/frontend/hoc/allow-override.tsx

```diff
@@ -29,8 +29,8 @@ function allowOverride<P extends Record<string, unknown>>(
      *
      */
     if (typeof window !== 'undefined') {
-      Component = (window as any).AdminJS?.UserComponents?.[name]
-        ?? (window as any).THEME?.Components?.[name]
+      Component = window.AdminJS?.UserComponents?.[name]
+        ?? window.THEME_COMPONENTS?.[name]
```

Comment:
```
Property 'THEME_COMPONENTS' does not exist on type 'Window & typeof globalThis'.
```
there seem to be some issues with global namespace

Reply:
Restored window as any to pass tests